source/docs/zero-to-robot/step-3/openmesh.rst

```diff
@@ -0,0 +1,231 @@
+
+.. warning:: The "Firewall" option configures the radio to emulate the field firewall. This means that you will not be able to deploy code wirelessly with this option enabled. This is useful for simulating blocked ports that may exist at competitions.
+
+Starting the Configuration Process
+----------------------------------
+
+.. image:: /docs/zero-to-robot/step-3/images/radio-programming/openmesh/start-config.png
+  :alt: At the bottom of the screen is instructions for completing the configuration process.
+
+Follow the on-screen instructions for preparing your wireless bridge, entering the settings the bridge will be configured with, and starting the configuration process. These on-screen instructions update to match the bridge model and operating mode chosen.
+
+Configuration Progress
+----------------------
+
+.. image:: /docs/zero-to-robot/step-3/images/radio-programming/openmesh/config-in-progress.png
+  :alt: The progress bar dialog while the configuration is working.
+
+Throughout the configuration process, the window will indicate:
+
+1. The step currently being executed.
+2. The overall progress of the configuration process.
+3. All steps executed so far.
+
+Configuration Completed
+-----------------------
+
+.. image:: /docs/zero-to-robot/step-3/images/radio-programming/openmesh/config-completed.png
+  :alt: A pop up dialog indicating the programming was successful.
+
+Once the configuration is complete:
+
+1. Press :guilabel:`OK` on the dialog window.
+2. Press :guilabel:`OK` on the main window to return to the settings screen.
+
+Configuration Errors
+--------------------
+
+.. image:: /docs/zero-to-robot/step-3/images/radio-programming/openmesh/config-errors.png
+  :alt: A error dialog pop up.
+
+If an error occurs during the configuration process, follow the instructions in the error message to correct the problem.
+
+Troubleshooting
+---------------
+
+- :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`.
+- Make sure you wait long enough that the power light has stayed solid for 10 seconds.
+- Make sure you have the correct network interface, and only one interface is listed in the drop-down.
+- Make sure your firewall is turned off.
+- Plug directly from your computer into the wireless bridge and make sure no other devices are connected to your computer via ethernet.
+- Ensure the ethernet is plugged into the port closest to the power jack on the wireless bridge.
+- If using an Operating System configured for languages other than US English, try using an English OS, such as on the KOP provided PC or setting the Locale setting to "en_us" as described on `this page <https://www.java.com/en/download/help/locale.html>`_.
+- Due to Unicode incompatibles, non-US Teams may face a configuration failure because of incorrect network interface reading. In that case, change the network adapter name to another name in English and retry.
```

Comment:
Typo
```suggestion
- Due to Unicode incompatibilities, non-US Teams may face a configuration failure because of incorrect network interface reading. In that case, change the network adapter name to another name in English and retry.
```